releng/org.eclipse.xtext.contributor/Xtext.setup

```diff
@@ -53,70 +53,116 @@
+      value="${p2.xtext.latest}"/>
+  <setupTask
+      xsi:type="setup:VariableTask"
+      name="p2.xtext.orbit"
+      value="http://download.eclipse.org/modeling/tmf/xtext/updates/orbit"/>
+  <setupTask
+      xsi:type="setup:VariableTask"
+      name="p2.mwe2"
+      value="http://download.eclipse.org/modeling/emft/mwe/updates/releases/${version.mwe2}"
+      label=""/>
   <setupTask
       xsi:type="setup:VariableTask"
-      name="mwe2.update.site"
-      value="http://download.eclipse.org/modeling/tmf/xtext/updates/composite/latest/"/>
+      name="p2.emf"
+      value="http://download.eclipse.org/modeling/emf/emf/updates/2.12"/>
   <setupTask
       xsi:type="setup:VariableTask"
-      name="emf.update.site"
-      value="http://download.eclipse.org/modeling/emf/emf/updates/2.12/"/>
+      type="URI"
+      name="p2.gef"
+      value="http://download.eclipse.org/tools/gef/updates/releases"/>
+  <setupTask
+      xsi:type="setup:VariableTask"
+      type="URI"
+      name="p2.xpand"
+      value="http://download.eclipse.org/modeling/tmf/xtext/updates/releases/xpand/R201406030414"/>
   <setupTask
       xsi:type="setup:VariableTask"
-      name="antlr-gen.update.site"
-      value="http://download.itemis.com/updates/releases/2.1.1/"/>
+      name="p2.antlr-gen"
+      value="http://download.itemis.com/updates/releases/2.1.1"/>
   <setupTask
       xsi:type="setup:VariableTask"
       type="URI"
-      name="draw2d.p2.repository"
-      value="http://download.eclipse.org/tools/gef/updates/legacy/releases/"/>
+      name="p2.draw2d"
+      value="http://download.eclipse.org/tools/gef/updates/legacy/releases"/>
   <setupTask
       xsi:type="setup:VariableTask"
       type="URI"
-      name="buildship.p2.repository"
-      value="http://download.eclipse.org/buildship/updates/e46/releases/2.x/"/>
+      name="p2.buildship"
+      value="http://download.eclipse.org/buildship/updates/e46/releases/2.x"/>
+  <setupTask
+      xsi:type="setup:VariableTask"
+      name="p2.m2e"
+      value="http://download.eclipse.org/technology/m2e/releases"/>
   <setupTask
       xsi:type="setup:VariableTask"
       name="p2.m2e.connector.jdt"
-      value="http://download.jboss.org/jbosstools/updates/m2e-extensions/m2e-jdt-compiler/"/>
+      value="http://download.jboss.org/jbosstools/updates/m2e-extensions/m2e-jdt-compiler"/>
   <setupTask
       xsi:type="setup:VariableTask"
       name="p2.m2e.connector.tycho"
-      value="http://repo1.maven.org/maven2/.m2e/connectors/m2eclipse-tycho/0.9.0/N/LATEST/"/>
+      value="http://repo1.maven.org/maven2/.m2e/connectors/m2eclipse-tycho/0.9.0/N/LATEST"/>
+  <setupTask
+      xsi:type="setup:VariableTask"
+      name="p2.webtools"
+      value="http://download.eclipse.org/webtools/repository/luna/">
+    <description>required by m2e</description>
+  </setupTask>
+  <setupTask
+      xsi:type="setup:VariableTask"
+      type="URI"
+      name="p2.findbugs"
+      value="http://findbugs.cs.umd.edu/eclipse"/>
```

Comment:
should we move to spotbugs? https://github.com/spotbugs/spotbugs

Reply:
In the past we had FindBugs. But actually I'm thinking about removing this feature. Such tools should be installed individual.
If we would have builds at Eclipse we could make use of their [SonarQube](https://sonar.eclipse.org/), like many other projects do. This would include FindBugs rules also.